### DISCUSSION TITLE
Install fail during sensor install

### DISCUSSION THREAD

**Discussion by cerberus10748** [Security-Onion-Solutions/securityonion#3982] [Unsupported Versions] [answered]
Hello, 
I am getting the following error after setting up some of the install on a distributed sensor.
bash: /root/manager_setup/securityonion/setup/so-setup: no such file or directory.

I have tried reinstalling and get the same error. I have installed sensors at other sites before and had success. Any ideas?

**Comment by cyb3rslack3r** [comment, marked as answer]
I had a similar error when I tried installing a new sensor on a distributed build after I upgraded all the other sensors to SO 2.3.40.  If you are using an older SO iso I would update it to whatever version the rest of the distribution is and see if that works

**Reply by cerberus10748** [reply]
That was it ! I had an old ISO and  I had already updated my manager. I downloaded the new ISO and it went right through.

**Comment by palichx** [comment]
try to remove this dir
/root/manager_setup/

and run so-sutup again

**Reply by cerberus10748** [reply]
Im a noob, how do I remoe that dir if I need to try this step?
